velox/exec/trace/CMakeLists.txt

```diff
@@ -13,7 +13,7 @@
 # limitations under the License.
 
 add_library(velox_query_trace_exec QueryMetadataWriter.cpp QueryTraceConfig.cpp
```

Comment:
I see the issue for the linking error. It's an issue with the monolithic library build and it's helper functions that now need to be used. Sorry we haven't communicated that well yet. #9948 took a while to merge so the targets here were added after CI was run but before it got merged, sorry for the inconvenience.

For targets that are not standalone executables like tests or fuzzers etc. you have to use `velox_add_library` instead of `add_library` and `velox_link_libraries` instead of `target_link_libraries` now.

Reply:
>you have to use velox_add_library instead of add_library and velox_link_libraries instead of target_link_libraries now.

@assignUser Do you mean that I don't need to link velox_query_trace_exec explicitly in `velox/benchmarks`, `velox/benchmarks/basic`, and `velox/dwio/dwrf/test` if I use `velox_add_library(velox_query_trace_exec...` and `velox_link_libraries(velox_query_trace_exec...` in `velox/exec/trace/CMakeList.txt`.

Reply:
@xiaoxmeng  please don't merge before this is fixed

Reply:
No, you still have to link the target where you are using it. When `VELOX_MONO_LIBRARY=OFF` everything behaves as if you were using the original functions so you still need those links.  (especially for the tests and benchmark as those are unaffected by the mono build)

Reply:
>For targets that are not standalone executables like tests or fuzzers etc. you have to use velox_add_library instead of add_library and velox_link_libraries instead of target_link_libraries now.

@assignUser Thanks for your suggestion, no more linking errors and ci passed.